CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ Specifically this means, that minor upgrades can contain **breaking changes**.
 
 # Change Log
 
+## 0.4.55
+
+**released 2023-02-20**
+* Upgrade go to v1.19.4 [#472](https://github.com/freiheit-com/kuberpult/pull/472)
+* Add documentation for config.json files [#475](https://github.com/freiheit-com/kuberpult/pull/475)
+* Improved error handling for release trains [#482](https://github.com/freiheit-com/kuberpult/pull/482)
+* Run kp in github actions [#484](https://github.com/freiheit-com/kuberpult/pull/484)
+* Display queued version in v2 release dialog [#491](https://github.com/freiheit-com/kuberpult/pull/491)
+* Bugfix: Add a check if the application version is available or not [#493](https://github.com/freiheit-com/kuberpult/pull/493)
+* Add createEnvironment http endpoint [#489](https://github.com/freiheit-com/kuberpult/pull/489)
+* Refactoring: Require return types in new ui [#492](https://github.com/freiheit-com/kuberpult/pull/492)
```

Comment:
We should only add points that are of interest for others, so I would trim this a bit:
```suggestion
* Upgrade go to v1.19.4 [#472](https://github.com/freiheit-com/kuberpult/pull/472)
* Add documentation for config.json files [#475](https://github.com/freiheit-com/kuberpult/pull/475)
* Improved error handling for release trains [#482](https://github.com/freiheit-com/kuberpult/pull/482)
* Display queued version in v2 release dialog [#491](https://github.com/freiheit-com/kuberpult/pull/491)
* Bugfix: Add a check if the application version is available or not [#493](https://github.com/freiheit-com/kuberpult/pull/493)
* Add createEnvironment http endpoint [#489](https://github.com/freiheit-com/kuberpult/pull/489)
```